### DISCUSSION TITLE
Fabric maven just doesn't download... at all (Intellij)

### DISCUSSION THREAD

**Discussion by kohtalee0910** [FabricMC/fabric-api#4028] [Mod Dev Support] [answered]
### Description

![image](https://github.com/user-attachments/assets/905e7649-30bf-4675-9715-4c4314736cfe)

every time it begins to download things from fabric mc, this happens, I tried turning off my firewall, reinstalling java, clearing my gradle cache, reinstalling intellij, nothing worked. Any idea what i'm doing wrong here?

**Comment by Linguardium** [comment]
try connecting to https://maven.fabricmc.net/ in a browser. if that works, something is blocking intellij
if it does not work, your ISP is likely blocking cloudflare or something
try setting the `maven.fabricmc.net` in settings.gradle to `maven2.fabricmc.net` to bypass cloudflare
if that works, call your ISP and have them fix their problem

**Reply by kohtalee0910** [reply]
it works but i am getting another error

**Comment by kohtalee0910** [comment]
![image](https://github.com/user-attachments/assets/df9da3ee-99ad-4f2f-bab7-71924690e895)
now im getting this error

**Comment by kohtalee0910** [comment, marked as answer]
nvm i fixed it, turns out maven was blocked

**Reply by kohtalee0910** [reply]
not cloudflare
